extensions/wikia/PortableInfobox/templates/PortableInfoboxItemImage.mustache

```diff
@@ -5,7 +5,7 @@
 <figure class="pi-item pi-image">
 	<a href="{{url}}" class="image image-thumbnail{{#isVideo}} video video-thumbnail small{{/isVideo}}"
 	   title="{{alt}}">
-		<img src="{{thumbnail}}" class="pi-image-thumbnail" alt="{{alt}}" width="{{{width}}}" height="{{{height}}}"
+		<img src="{{thumbnail}}" class="pi-image-thumbnail" alt="{{alt}}" width="100%" height="{{{height}}}"
```

Comment:
could you invert it - set width to 300px so correct thumbnail asset width and set in in CSS to `width: 100%`?